casr/tests/tests.rs

```diff
@@ -4562,3 +4564,977 @@ fn test_casr_cluster_d_python() {
 
     let _ = std::fs::remove_dir_all(&paths[1]);
 }
+
+#[test]
+#[cfg(target_arch = "x86_64")]
+fn test_casr_js() {
+    let test_dir = abs_path("tests/tmp_tests_casr/test_casr_js");
+    let test_path = abs_path("tests/casr_tests/js/test_casr_js.js");
+    let _ = std::fs::remove_dir_all(&test_dir);
+    let Ok(node_path) = which::which("node") else {
+        panic!("No node is found.");
+    };
+
+    let output = Command::new(*EXE_CASR_JS.read().unwrap())
+        .args(["--stdout", "--", &node_path.to_str().unwrap(), &test_path])
+        .output()
+        .expect("failed to start casr-js");
+
+    assert!(
+        output.status.success(),
+        "Stdout {}.\n Stderr: {}",
+        String::from_utf8_lossy(&output.stdout),
+        String::from_utf8_lossy(&output.stderr)
+    );
+
+    let report: Result<Value, _> = serde_json::from_slice(&output.stdout);
+    if let Ok(report) = report {
+        let severity_type = report["CrashSeverity"]["Type"].as_str().unwrap();
+        let severity_desc = report["CrashSeverity"]["ShortDescription"]
+            .as_str()
+            .unwrap()
+            .to_string();
+
+        assert_eq!(10, report["Stacktrace"].as_array().unwrap().iter().count());
+        assert_eq!(severity_type, "NOT_EXPLOITABLE");
+        assert_eq!(severity_desc, "Error");
+        assert!(report["CrashLine"]
+            .as_str()
+            .unwrap()
+            .contains("test_casr_js.js:3:15"));
+    } else {
+        panic!("Couldn't parse json report file.");
+    }
+    let _ = std::fs::remove_dir_all(test_dir);
```

Comment:
Do not remove out  directory at the end. We remove tmp directories at the beginning.  Fix this in other tests you added too.

Reply:
Done